### PR TITLE
Collect metrics from Sidekiq when installed

### PR DESCRIPTION
**Pull request by franzliedke**
Following in the footsteps of #5, this automatically collects metrics from Sidekiq workers, as previously tested in one of our internal applications.

### CODE REVIEW

File: lib/telegraf/sidekiq.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        # The "enqueued_at" key is not present for scheduled jobs.
+        # See https://github.com/mperham/sidekiq/wiki/Job-Format.
+        if job.key?('enqueued_at')
+          enqueued_at = ::Time.at(job['enqueued_at'].to_f).utc
+          values[:queue_ms] = (job_start - enqueued_at) * 1000 # milliseconds
+        end
+
+        # The "at" key is only present for scheduled jobs.
+        tags[:type] = 'scheduled_job' if job.key?('at')
```
Comment (franzliedke) [inline lib/telegraf/sidekiq.rb:70]: Difference to our internal version: I do not store this timestamp (or rather the difference from it, called "delay"), as I could not come up with a use case at this time.

File: lib/telegraf/sidekiq.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      end
+
+      def call(worker, job, queue)
+        job_start = ::Time.now.utc
+
+        tags = {
+          **@tags,
+          type: 'job',
+          errors: true,
+          retry: job.key?('retried_at'),
```
Comment (franzliedke) [inline lib/telegraf/sidekiq.rb:53]: I tried very hard to test retries, but that is not possible, as far as I could tell (and what Sidekiq's author commented in a question on StackOverflow).

Comment (jgraichen) [reply lib/telegraf/sidekiq.rb:53]: Fine for me.

**Comment by jgraichen** [comment]
Looks really good!

**Pull request merged**